internal/commitinfo/commitinfo.go

```diff
@@ -68,11 +68,11 @@ func ParseCommitInfo(commitMessage string) (CommitInfo, error) {
 	}
 
 	matches := parseCommitInfoFromCommitMessageRegex.FindStringSubmatch(convInfo.Message)
-	if matches == nil {
```

Comment:
Don't we still need a check on just `matches == nil` ? With the extra case we might get to the bottom of this method with `matches == nil`?

Same in the case that `matches != nil` but non of the `service`, `artifactID` etc. checks match, right?

Reply:
No, we don't care about the commit message parsing if there is a "Release-intent". We can read it all from fields, so there isn't a real reason for trying to parse the message, other than backward compatibility.

Reply:
Ok. Makes sense 👍